frontend/src/services/eventHandlers/gameHandler.js

```diff
@@ -76,6 +76,8 @@ export default class GameHandler {
                 );
                 EventBus.$emit(Event.CORRECT_GUESS, player);
             }
+        } else if (payload.gameMasterAPI === "showResults") {
+            EventBus.$emit(Event.GAME_OVER, payload);
```

Comment:
Should this be a state instead of an event?

Reply:
Good idea, I'll change that